src/js/cli.js

```diff
@@ -34,10 +34,24 @@ export type CLIOptsType = {
   args: string[],
 };
 
-export function createBanner(): string {
+
+function getClojureScriptVersionString(): string {
   // $FlowFixMe: we know for sure this file will exist.
-  return `Lumo ${lumoVersion}
-ClojureScript ${lumo.load('clojurescript-version')}
+  return `ClojureScript ${lumo.load('clojurescript-version')}`;
+}
+
+function getNodeVersionString(): string {
+  return `Node ${process.versions.node}`;
```

Comment:
can we print `Node.js v7.8.0`?
change Node -> Node.js
change `process.versions.node` -> `process.version`